src/models/accounts.js

```diff
@@ -44,19 +44,25 @@ function * getAccounts () {
 function * getAccount (name, requestingUser) {
   log.debug('fetching account name ' + name)
 
-  const canExamine = requestingUser &&
-    (requestingUser.name === name || requestingUser.is_admin)
-  const account = yield db.getAccount(name)
-  if (!account) {
-    throw new NotFoundError('Unknown account')
-  } else if (account.is_disabled &&
+  let canExamine = false
+  let account
+  if (!requestingUser) { // anonymous request
+    account = {id: name, name: name}
+  } else { // authenticated request
+    canExamine = requestingUser.name === name || requestingUser.is_admin
+    account = yield db.getAccount(name)
+    if (!account) {
+      throw new NotFoundError('Unknown account')
+    } else if (account.is_disabled &&
       (requestingUser && !requestingUser.is_admin)) {
-    throw new UnauthorizedError('This account is disabled')
+      throw new UnauthorizedError('This account is disabled')
+    }
+
+    // TODO get rid of this when we start using biginteger math everywhere
+    account.balance = Number(account.balance).toString()
+    delete account.password_hash
```

Comment:
I agree although I did not add this line. It is already in master. 

In addition, it seems to be redundant since `converter.convertToExternalAccount(...)` in [line 66](https://github.com/interledgerjs/five-bells-ledger/blob/f4055ea73bad19211c96a96d0619acce93fc4d66/src/models/accounts.js#L66) remove this property too.

Does anybody see a reason why we should not drop this line?